app/src/main/dist/config/crate.yml

```diff
@@ -216,7 +216,7 @@
 
 # Node names are generated dynamically on startup, so you're relieved
 # from configuring them manually. You can tie this node to a specific name:
-#node.name: "Piz Buin"
+#node.attr.name: "Piz Buin"
```

Comment:
This is still `node.name` without `attr`

Reply:
`node.name` is still the same. https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html#node.name